src/flux/course.coffee

```diff
@@ -0,0 +1,35 @@
+_ = require 'underscore'
+
+{TaskActions, TaskStore} = require './task'
+{CrudConfig, makeSimpleStore, extendConfig} = require './helpers'
+
+CourseConfig =
+
+  loadPractice: (courseId) ->
+    @_local[courseId] ?= {}
+    @_local[courseId]['practices'] ?= []
+
+  loadedPractice: (obj, courseId) ->
+    @_local[courseId]['practices'].push(obj)
```

Comment:
there is only ever one current practice per course per student.  We keep track of the old ones for reports, etc, but whenever a `POST` is made to `/api/courses/1/practice`, we shut down the old one and make a new current one.